src/graphql/mutation.rs

```diff
@@ -32,13 +32,16 @@ graphql_object!(MutationRoot: RequestContext as "Mutation" |&self| {
         project: ProjectRequest,
         token: String
     ) -> FieldResult<Tokenized<Project>> {
+        use crate::resolvers::{user, project};
```

Comment:
I'm not a huge fan of embedding imports in code. Why not import at top?

Reply:
Idk, tbh I kinda get uneasy about having a bunch of names in the file scope, but I don't have a good reason for or against, it's just personal preference and conditioning from writing in C I guess.

Reply:
Let's stick to imports on top. I like being able to see what's being imported in one place. It's completely subjective, but we might as well stick to one system.

Reply:
Ok, fixed!